templates/tfengine/components/cicd/configs/tf-apply.yaml

```diff
@@ -12,29 +12,22 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-include {
-  path = find_in_parent_folders()
-}
+# Note: Terraform version used in the automation is 0.12.29.
+{{- $cft := "gcr.io/cloud-foundation-cicd/cft/developer-tools@sha256:47db3e958fbaa0d95881cf99501ebf4522a261ddc68d9566b7cf70e26cf7cddb"}}
 
-dependency "networks" {
-  config_path = "../../example-prod-networks/resources"
-  mock_outputs = {
-    bastion_service_account = "mock-sa"
+timeout: 21600s
 
-  }
-}
+steps:
+  - name: "{{$cft}}"
+    entrypoint: terraform
+    args: ["version"]
+    id: Terraform version
 
-dependency "project" {
-  config_path = "../project"
-  mock_outputs = {
-    project_id = "mock-project"
+  - name: "{{$cft}}"
+    entrypoint: bash
+    args: ["./cicd/configs/run.sh", "-a", "plan", "-a", "apply -auto-approve"]
+    dir: "${_TERRAFORM_ROOT}"
+    id: Apply
 
-  }
-}
-
-inputs = {
-  bastion_service_account = "${dependency.networks.outputs.bastion_service_account}"
-
-  project_id = "${dependency.project.outputs.project_id}"
-
-}
+substitutions:
```

Comment:
Nit: I think it's better to have substitutions right above the "steps:" block, but if it's trouble to change it, I'm okay leaving it down here too.